### ISSUE TITLE
Update FTP credentials form design

### ISSUE THREAD

**Issue by melchoyce** [obenland/shiny-updates#100]
Current designs:

<img width="360" alt="ftp-credentials_720" src="https://cloud.githubusercontent.com/assets/2846578/15256295/f34c7ed2-190e-11e6-86a9-e8a8c72578f8.png">
<img width="275" alt="ftp-credentials-with-ssh_720" src="https://cloud.githubusercontent.com/assets/2846578/15256294/f347784c-190e-11e6-9abb-4521e54a998a.png">

These could use with just a little bit of tweaking to improve hierarchy and spacing. I'll work on that a bit and then post up some designs here.

**Comment by melchoyce** [comment]
Some updates to the form in https://core.trac.wordpress.org/ticket/34376. (Connection Type radios at the very top of the form instead of the bottom)

**Comment by swissspidy** [comment]
Note that the core ticket mentioned above has now been fixed, see https://core.trac.wordpress.org/changeset/37467.

What should be done now as part of Shiny Updates?

**Comment by melchoyce** [comment]
Thinking a couple things:
1. Update modal style to bring more in-line with other wp-admin modals — like including an "x" for closing (even though there's also a cancel button)
2. Make "Proceed" primary
3. Also consider changing "proceed" to something more descriptive — maybe connect? Proceed might be okay, it's just such a weird-sounding word. Maybe switch the "proceed" in both the description and the button to "continue."
4. "This password will not be..." could stand to be decreased in size just a little and made a lighter grey, to match descriptions on the settings pages. At the very least, the line-height should decrease. 
5. "Authentication Keys" is at the same level of hierarchy as the field labels. It should probably be bigger, since it's a header above the labels.

I can mock these up these changes if you'd like.

**Comment by melchoyce** [comment]
Here's my recommendation:

![shiny-update-ftp](https://cloud.githubusercontent.com/assets/2846578/15518821/a66ceae0-21cc-11e6-9c98-791badd5b30d.jpg)

Ideally, we'd switch the username/password labels based on whether someone has ftp/ssh selected.

**Comment by melchoyce** [comment]
Mockup + PSD: https://cloudup.com/cqzFnU7nnId

**Comment by afercia** [comment]
Worth considering the whole "Authentication Keys" block appears only when selecting the radio button SSH2 (which is not even shown is SSH is not available). So, selecting the radio button on top of the form and having something appear on the bottom of the form wouldn't be ideal for accessibility.
This was actually part of the job done in the mentioned changeset and ticket, see above. Please don't change the order of the elements in the form :)

**Comment by melchoyce** [comment]
Sorry, guess I was confused about that trac ticket — that's what I thought it was doing, moving the radio buttons to the top.

**Comment by afercia** [comment]
No worries it's a bit difficult to test the interaction here, actually you need a small plugin to "fake" the SSH availability and see the real interaction.

**Comment by melchoyce** [comment]
Updated to switch order: https://cloudup.com/cnvj5NqMefR

**Comment by melchoyce** [comment]
Additionally: the modal is really tall for mockup purposes, but would ideally be whatever height we normally max modals out at. Any content not visible because of height would be scrollable in the modal, like we do in others.

**Comment by swissspidy** [comment]
@afercia Which plugin are you using for that?

**Comment by afercia** [comment]
"Force FTP" by Dominik Schilling :)

**Comment by swissspidy** [comment]
Thank you very much @melchoyce, I like it a lot.

In #164, I fixed the non-functioning filesystem credentials dialog after https://core.trac.wordpress.org/changeset/37467. I also tweaked the CSS a little

Unfortunately `request_filesystem_credentials()` isn't filterable, which means any other suggested changes like making "Proceed" a primary button and adding a Cancel link need to happen on core directly, and not in this plugin.
